src/uberjob/_registry.py

```diff
@@ -65,8 +65,7 @@ def source(self, plan: Plan, value_store: ValueStore) -> Node:
         validation.assert_is_instance(plan, "plan", Plan)
         validation.assert_is_instance(value_store, "value_store", ValueStore)
         stack_frame = get_stack_frame()
-        node = plan.call(source)
-        node.stack_frame = stack_frame
+        node = plan._call(stack_frame, source)
```

Comment:
Performance optimization -- the previous code was computing an extra symbolic traceback.